tests/tox.ini

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
We will probably need folders for each workflow and multiple folders for some of the workflows with different tests